DIPs/DIP1337.md

```diff
@@ -0,0 +1,85 @@
+# Tail Call Optimization requirement
+
+| Section         | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | 1337                                                            |
+| Author:         | Dietrich Daroch (name dot surname on gmail)                     |
+| Implementation: | (links to implementation PR if any)                             |
+| Status:         | -                                                               |
+
+
+
+## Abstract
+
+Add an attribute that forces tail call optimization.
+
+
+### Links
+
+Optional sections containing links to existing discussions, research papers or any
+other supplemental materials.
+
```

Comment:
Oh crap, I ended up using the JPL's C guidelines directly on the rationale part.